backend/infrastructure/__init__.py

```
@@ -0,0 +1,4 @@
+"""Infrastructure layer.
+
+外部システム統合（DB, LLM, Tools）を実装します。
```

Ruff (0.14.11)

Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)